packages/01-host/webpack.config.js

```diff
@@ -46,8 +52,8 @@ module.exports = {
       },
       shared: ["react", "react-dom","react-router-dom"]
     }),
-    new HtmlWebpackPlugin({
-      template: "./public/index.html"
-    })
+    new CopyPlugin([
```

Comment:
Using the copy plugin to ensure serve.json is in the dist directory (so that cors works)